packages/ses-integration-test/test/sanity.test.js

```diff
@@ -0,0 +1,17 @@
+/* global Compartment */
+import test from "tape";
+
+// eslint-disable-next-line import/no-unresolved
+import * as SES from "ses";
```

Comment:
I’m still unclear about which cases `import name` works but `import * as name` does not. It’s the case that a ESM module converted to CJS can’t express this?

Reply:
Not really.  Again, it's the test framework, which magically deletes this line when using the UMD build (which creates an `SES` global containing `SES.lockdown`).  This test needs to be written this way, or it can't be transpiled for all the different bundlers.